experimental/PropertyDDS/packages/property-inspector-table/src/InspectorTable.tsx

```diff
@@ -405,24 +411,18 @@ class InspectorTable extends React.Component<WithStyles<typeof styles> & IInspec
 
     public render() {
       const {
-        childGetter,
         classes,
-        columns,
         currentUrn,
         data,
         deleteRepo,
         expired,
-        followReferences,
         getRepoExpiry,
         height,
         isV1Urn,
-        nameGetter,
         repositoryUrn,
-        rowIconRenderer,
         searchBoxProps,
         setRepoExpiry,
         width,
-        activeRepositoryGuid,
```

Comment:
Do these removals imply that someone might be passing in useless props that we could trim?  And/or the props interface might want trimming too?